spec/graphql/queries/todos/get_todo_by_category_ids.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Queries
+  module Todos
+    RSpec.describe Todos, type: :request do
+      describe '.resolve' do
+        let(:category_1) do
+          create(:category)
+        end
```

Comment:
一行で済む場合はこんな感じで一行で書くことが多いです！
```suggestion
        let(:category_1) { create(:category) }
```

Reply:
はい、これで書くならもっとcleanになりますね！ありがとうございます！😁